src/functions.ts

```diff
@@ -1664,112 +1584,124 @@ ${main_content}
         });
 
         cards.forEach(c => {
-            this.remove(_list, c);
+            functions.remove(_list, c);
         });
 
         return cards;
-    }
+    },
 
     /**
      * Creates and returns a jade golem with the correct stats and cost for the player
      * 
-     * @param {Player} plr The jade golem's owner
+     * @param plr The jade golem's owner
      * 
-     * @returns {Card} The jade golem
+     * @returns The jade golem
      */
-    createJade(plr) {
+    createJade(plr: Player): Card {
         if (plr.jadeCounter < 30) plr.jadeCounter += 1;
         const count = plr.jadeCounter;
         const mana = (count < 10) ? count : 10;
 
-        let jade = new game.Card("Jade Golem", plr);
+        let jade = new Card("Jade Golem", plr);
         jade.setStats(count, count);
         jade.mana = mana;
 
         return jade;
-    }
+    },
 
     /**
      * Imports the config from the `path` specified.
      *
-     * @param {string} path The path to import from.
+     * @param path The path to import from.
      *
-     * @returns {boolean} Success
+     * @returns Success
      */
-    importConfig(path) {
+    importConfig(path: string): boolean {
+        // @ts-expect-error - Typescript doesn't expect the config to be empty, but it gets repopulated immediately anyway.
         game.config = {};
+        path = path.replace("/dist", "");
 
-        require("fs").readdirSync(path, { withFileTypes: true }).forEach(file => {
+        fs.readdirSync(path, { withFileTypes: true }).forEach(file => {
             let c = `${path}/${file.name}`;
 
             if (file.name.endsWith(".json")) {
-                let f = require(c);
+                let f = import(c);
 
                 game.config = Object.assign({}, game.config, f);
             }
-            else if (file.isDirectory()) this.importConfig(c);
+            else if (file.isDirectory()) functions.importConfig(c);
         });
 
         game.doConfigAI();
 
         return true;
-    }
+    },
 
     /**
      * USE @see {@link importCards} INSTEAD. Imports all cards from a folder.
      * 
-     * @param {string} path The path
+     * @param path The path
      * 
-     * @returns {boolean} Success
+     * @returns Success
      */
-    _importCards(path) {
-        require("fs").readdirSync(path, { withFileTypes: true }).forEach(file => {
+    _importCards(path: string): boolean {
+        fs.readdirSync(path, { withFileTypes: true }).forEach(file => {
             let p = `${path}/${file.name}`;
 
-            if (file.name.endsWith(".js")) {
-                let f = require(p);
-                
-                game.cards.push(f);
+            if (file.name.endsWith(".ts")) {
+                let c = require(p);
```

Comment:
This line is causing problems. `require` does not work in es modules, but dynamic `import` returns a promise. I am kind of stuck here.